Jenkinsfile

```diff
@@ -99,5 +101,26 @@ pipeline {
         """
       junit testResults: "logs/junit.xml", keepLongStdio: false
     }
+    success {
+      script {
+        if( env.UMB_MESSAGE == true || env.UMB_MESSAGE == "true" ){
+          sendCIMessage \
+            providerName: 'Red Hat UMB', \
+            overrides: [ topic: 'VirtualTopic.qe.ci.jenkins' ], \
+            failOnError: false, \
+            messageType: 'Tier1TestingDone', \
+            messageProperties: '''
+              TOOL=ocs-ci
```

Comment:
ocs-ci-dev ?

Reply:
Not sure what you mean here

Reply:
I was suggesting the TOOL to be changed as ocs-ci-dev but I guess it doesn't matter since CI_MESSAGE is what gets parsed.